### PR TITLE
React Redux Capstone Project

### PR DESCRIPTION
**Pull request by zainsadaqat**
## In this Pull Request of my React Redux Capstone Project, I've done the following:

- Create professional project `directories structure`
- Use `functional components`
- Use `props`
- Use `Hooks`
- Use react functional `life cycle` function
- Use Custom `CSS` styles
- Write `Test` cases
- Use `store`, `actions`, and `reducers` in React
- Use `react-routers`
- Follow `Behance Design`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,67 @@
-# Getting Started with Create React App
+# React Redux Capstone Project (Bank Holidays in United Kingdom)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+> In this project, we will be working with the real live data from the UK Govt API. We are going to build a web application that tells us the holidays in UK calender. Bank holidays in England and Wales, Scotland and Northern Ireland
```
Comment (Mihndim2020) [inline README.md:4]: - [x] Please as it is a requirement for this project, please kindly deploy your application using `Heroku`, `Netlify` or `GitHub Pages` (or any platform of your choice).
- [x] Please add a link to your  deployed project in this `README` file.

File: README.md [hunk 2]
```diff
@@ -1,70 +1,67 @@
-# Getting Started with Create React App
+# React Redux Capstone Project (Bank Holidays in United Kingdom)
```
Comment (Mihndim2020) [inline README.md:2]: - [x] Please as it is a requirement for this project, kindly add a presentation video as specified in the screenshot below of the project requirement

video

- [x] Please add a link to the video presentation in your `Readme` file.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: Your project is running

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (zainsadaqat) [reply README.md:2]: ### I completely forgot about making a video. Let me create it. 😃

**Comment by omar-labana** [comment]
Hi @zainsadaqat, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by zainsadaqat** [comment]
# Thank you so much! @omar-labana for approving my pull request. 🥳

**Pull request merged**